tests/phpt/serialize_broken_class.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Test that requiring a broken class (with a parser error) will not explode during serialization
+--FILE--
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Tests;
+
+use Sentry\ClientBuilder;
+use Sentry\Event;
+use Sentry\Integration\FatalErrorListenerIntegration;
+use Sentry\Options;
+use Sentry\Serializer\Serializer;
+use Sentry\State\Hub;
+use Sentry\Tests\resources\BrokenClass;
+use Sentry\Transport\TransportInterface;
+
+$vendor = __DIR__;
+
+while (!file_exists($vendor . '/vendor')) {
+    $vendor = dirname($vendor);
+}
+
+require $vendor . '/vendor/autoload.php';
+
+function testSerialization($value) {
```

Comment:
Any reason for this method to exists instead of directly instantiating the serializer and calling it?

Reply:
Yes, it's needed to reproduce the issue, which is triggered by the backtrace serialization.

Reply:
Can you please add a comment explaining why it's needed and/or link to the issue? Otherwise it's not clear and in the future someone may think that it's useless like I did